stanford_soe_r25.info

```diff
@@ -0,0 +1,10 @@
+name = R25 Room Reservations for Stanford SoE Intranet
+description = Interface for room reservations for 25Live/R25 for the Stanford SoE Intranet
+core = 7.x
+package = Stanford
+version = 7.x-1.0-alpha1
```

Comment:
This should be dev until the release. Which you are planning to do right after this gets merged I assume.